### DISCUSSION TITLE
A dotenv Error to a newbie

### DISCUSSION THREAD

**Discussion by jonathanwasaki** [goatandsheep/react-native-dotenv#202] [General]
I am getting this dotenv error:
**try to import dotenv variable "server url" which is not defined in any variables**

https://drive.google.com/file/d/1oPVEpWl1H9tWbt7BUOzWPLhbR2gbeYPR/view?usp=sharing

https://drive.google.com/file/d/1FrKMFobuUEtqS6cjOAwUs1-d5I33qQsK/view?usp=sharing

This is what is inside my **.env file**
https://drive.google.com/file/d/166bRD83KRNYDU9ZDpRvHpuQuhptDn1Gk/view?usp=sharing

The error appears to be here
https://drive.google.com/file/d/1j1NrKIfDXRFv_4lkg2jEInIB6T7XtfTi/view?usp=sharing

Am trying to run this repo
https://github.com/steniowagner/mindcast-server

This is what I have been doing
https://drive.google.com/drive/folders/11gZMOqawo_y7EXZ9g2CsQr7-rqLoGwFQ?usp=sharing

I need some help on how to solve the error

**Comment by goatandsheep** [comment]
look at the migration guide in the wiki :) there have been some big changes since that version of react-native-dotenv
